### DISCUSSION TITLE
Grass that needs to be cut

### DISCUSSION THREAD

**Discussion by zhanghengyuan757** [Melledy/Grasscutter#2070] [Ideas]
我不清楚应该从Grasscutter resource 还是Grasscutter本身来提这些issues：
1*、强烈建议完善角色的治疗功能，没有技能治疗，感觉缺失很多，即便是打开无敌指令。（很期待）
2、角色技能不产生元素微粒或很少，这一点很建议做出调整。
3*、希望新增特性 重温原神的魔神任务剧情。（很期待）
4、希望新增特性 接取每日任务。
5、希望新增特性 游戏内的秘境挑战：周本秘境、圣遗物秘境、武器突破秘境、天赋书秘境，等等。
6*、希望新增特性 深渊9-12层。（很期待）
7、希望新增特性 重温原神的限时活动。

原神的火热，一部分归功于其持续的更新与运营。
Grasscutter割草机填补了玩家一部分长草期，但是上面问题让人非常长草。

**Comment by ghost** [comment]


**Reply by zhanghengyuan757** [reply]
Thanks! finally find the organiztion! https://ptb.discord.com/channels/965284035985305680

**Reply by zhanghengyuan757** [reply]
Thanks! Finally found the organization.
